.github/dependabot.yml

```diff
@@ -27,14 +27,16 @@ updates:
       - dependency-name: com.vackosar.gitflowincrementalbuilder:gitflow-incremental-builder
       - dependency-name: org.jboss.logging:*
       - dependency-name: org.jboss.logmanager:*
-      - dependency-name: org.apache.logging.log4j:log4j-api
+      - dependency-name: org.apache.logging.log4j:*
+      - dependency-name: org.slf4j:*
       - dependency-name: org.ow2.asm:*
       - dependency-name: org.glassfish:jakarta-el
       - dependency-name: com.google.cloud.tools:jib-core
       # Quarkus
       - dependency-name: io.quarkus.gizmo:gizmo
       - dependency-name: io.quarkus.http:*
       - dependency-name: io.quarkus:quarkus-fs-util
+      - dependency-name: io.quarkus:quarkus-spring-*-api
```

Comment:
@geoand I think you want the Spring APIs updated? Some are already outdated. BTW, we should probably have used a separate groupId for them.

Reply:
Sure yeah, we can track them